examples/policies/l3/requires/endpoints.json

```diff
@@ -0,0 +1,9 @@
+[{
+    "labels": [{"key": "name", "value": "l3-rule"}],
+    "endpointSelector": {"matchLabels": {"role":"forntend"}},
```

Comment:
```suggestion
    "endpointSelector": {"matchLabels": {"role":"frontend"}},
```

Reply:
Well actually the yaml example uses `env: prod` instead

Reply:
Thanks! Fixed both issues.